### PR TITLE
[Assignment 3] Add tests and externalize server initialization

### PR DESCRIPTION
**Pull request by zerok**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -10,19 +9,24 @@ var Version = require('./version');
 var internals = {};
```
Comment (chyld) [inline lib/index.js]: don't need internals or internals.init. you can just delete it. it's not used.

File: lib/start.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+var Server = require('./index');
```
Comment (chyld) [inline lib/start.js]: start.js should be in the root directory

**Comment by zerok** [comment]
@chyld both corrected. Thanks for the feedback :)

**Comment by zoe-1** [comment]
@zerok checkout @hueniverse comments made in PR #80. 
There are many changes described in that PR that you need to implement. 
Check it out :-)

**Pull request closed without merging**